traffic-advice.bs

```diff
@@ -0,0 +1,265 @@
+<pre class="metadata">
+Title: Traffic Advice
+Shortname: traffic-advice
+Status: DREAM
+Repository: buettner/private-prefetch-proxy
+Editor: Jeremy Roman, Google https://www.google.com/, jbroman@chromium.org
+Abstract: A proposal to allow site owners to advice prefetch proxies and other agents to disallow traffic.
+Markup Shorthands: css no, markdown yes
+Assume Explicit For: yes
+Complain About: accidental-2119 yes, missing-example-ids yes
+Indent: 2
+Boilerplate: omit conformance
+</pre>
+<pre class="biblio">
+{
+    "HTTP-SEMANTICS": {
+        "aliasOf": "RFC7231"
+    },
+    "WELL-KNOWN": {
+        "aliasOf": "RFC8615"
+    }
+}
+</pre>
+
+Introduction {#intro}
+=====================
+
+*This section is non-normative.*
+
+Publishers may wish not to accept traffic from private prefetch proxies and other sources other than direct user traffic, for instance to reduce server load due to speculative prefetch activity.
+
+We propose a well-known "traffic advice" resource, analogous to `robots.txt` (for web crawlers), which allows an HTTP server to declare that implementing agents should stop sending traffic to it for some time.
+
+
+Infrastructure {#infra}
+=======================
+
+This specification depends on the Infra Standard. [[INFRA]]
+
+This specification uses terminology from the Fetch, HTTP, HTML and URL standards. [[FETCH]] [[HTTP-SEMANTICS]] [[HTML]] [[URL]]
```

Comment:
I might omit this as it basically duplicates (incompletely) https://jbroman-spec-drafts.glitch.me/traffic-advice.html#index-defined-elsewhere

Reply:
Fair enough.